### DISCUSSION TITLE
Extension Update Chrome

### DISCUSSION THREAD

**Discussion by r0senove** [night/betterttv#6461] [Q&A] [answered]
Hey, how long did it takes to get latest version on extension chrome?
because, right from i start this new discussion, latest release on github is 7.8.9 (19 hours ago), but at chrome extension is still 7.5.8.

thank you.

**Comment by night** [comment, marked as answer]
The latest release is 7.5.9, not 7.8.9. Updates take as long as the submission review process takes with Chrome, Edge, and Firefox.
